Astro CLI 1.7 Release Notes and New Docs (#1432)
astronomer/docs: astro/cli/astro-context-list copy.md

---
sidebar_label: "astro organization list"
title: "astro organization list"
id: astro-organization-list
description: Reference documentation for astro organization list.
hide_table_of_contents: true
---

> View a list of Organizations that you have access to. An Organization will appear on this list if you have been given access to it be a System Admin.

Reviewer: ```suggestion
View a list of Organizations that you can access. Only the Organizations that you are assigned System Admin permissions appear on this list.
```

## Usage

```sh
astro organization list
```

> ## Related Commands

Reviewer: ```suggestion
## Related commands
```
